src/issuance.rs

```diff
@@ -1212,12 +1221,12 @@ mod tests {
         .unwrap();
 
         let sighash: [u8; 32] = bundle.commitment().into();
-        let signed = bundle.prepare(sighash).sign(rng, &isk).unwrap();
+        let signed = bundle.prepare(sighash).sign(&isk).unwrap();
         let prev_finalized = &HashSet::new();
 
         assert_eq!(
             verify_issue_bundle(&signed, random_sighash, prev_finalized).unwrap_err(),
-            IssueBundleInvalidSignature(InvalidSignature)
+            IssueBundleInvalidSignature(schnorr::Error::new())
```

Comment:
Here, I'm using the schnorr::Error without any source details. Suggestions appreciated.

Reply:
Done